spec/beaker-puppet/install_utils/puppet5_spec.rb

```diff
@@ -199,6 +200,19 @@ def run_shared_test_steps
         run_shared_test_steps()
       end
 
+      it 'AIX: fetches the file & runs local install' do
+        @platform = 'aix'
+        @version = '7.2'
```

Comment:
Could you add a separate test for 7.1, as we only do the `ignoreos` part on 7.2.